### PR TITLE
FIX: data dictionary for distribution grids

### PR DESCRIPTION
**Pull request by keatsig**
This PR modifies data dictionary for unbalanced distribution grid cases (supports openDSS files parsed by PowerModelsDistribution.jl) to account for the three-phase parameters.

### CODE REVIEW

File: src/plots/plot_processing.jl [hunk 1]
```diff
@@ -0,0 +1,110 @@
+"Modify distribution grid data dictionary"
+function distr_data(case::Dict{String,<:Any})
+    # create new dict
+    case_dist = Dict{String, Any}("bus"=>Dict{String, Any}(),
+        "gen"=>Dict{String, Any}(),
+        "branch"=>Dict{String, Any}(),
+        "switch"=>Dict{String, Any}(),
+        "transformer"=>Dict{String, Any}()
+        )
```
Comment (noahrhodes) [inline src/plots/plot_processing.jl:9]: This does not copy the metadata of the grid, like the name, per unit etc.  I would make a deep-copy of the old network and replace these component dictionaries with empty ones
```julia
case_dist = deepcopy(case)
for comp_type in ["gen","branch","bus","switch","transformer"]
    case_dist[comp_type] = Dict{String,Any}()
end
```

**Review by noahrhodes — CHANGES_REQUESTED** [review]
Just one comment on the code! it looks good overall.

Can you also add some tests?  You can add a small distribution system and run the powerplot with it to check that code is run.

**Review by noahrhodes — APPROVED** [review]

**Comment by noahrhodes** [comment]
Is PMD required as a dependency? (are any PMD functions called by Power Plots?).  This can make a big difference in allowing parallel precompilation of PowerPlots and PMD when updating the packages.

**Comment by noahrhodes** [comment]
TODO:

- [ ] Update Changelog
- [ ] Update documentation

**Comment by noahrhodes** [comment]
@keatsig I updated the changelog.  I this ready to merge?

**Comment by keatsig** [comment]
Yes, it is ready to merge.
________________________________

**Review by noahrhodes — APPROVED** [review]

**Pull request merged**